src/crimson/os/seastore/onode_manager/staged-fltree/node.cc

```diff
@@ -1809,31 +1809,30 @@ LeafNode::erase(context_t c, const search_position_t& pos, bool get_next)
   LOG_PREFIX(OTree::LeafNode::erase);
   assert(!pos.is_end());
   assert(!impl->is_keys_empty());
-  Ref<Node> this_ref = this;
   DEBUGT("erase {}'s pos({}), get_next={} ...",
          c.t, get_name(), pos, get_next);
 
   // get the next cursor
-  return seastar::now().then([c, &pos, get_next, this] {
+  return seastar::now().then([this, c, &pos, get_next] {
```

Comment:
why order matters here? also, i'd prefer put `this` and the end of the capture list to emphasis the regular variables.